src/query/mod.rs

```diff
@@ -19,14 +19,21 @@
 //! assert!(read_query.is_ok());
 //! ```
 
+#[cfg(any(test, feature = "chrono_timestamps"))]
+extern crate chrono;
+#[cfg(any(test, feature = "chrono_timestamps"))]
+use chrono::prelude::{DateTime, TimeZone, Utc};
+#[cfg(any(test, feature = "chrono_timestamps"))]
+use std::convert::TryInto;
+
 pub mod read_query;
 pub mod write_query;
 
 use std::fmt;
 
 use crate::{Error, ReadQuery, WriteQuery};
 
-#[derive(PartialEq)]
+#[derive(PartialEq, Debug)]
```

Comment:
need the Debug trait for the tests to work properly

Reply:
`Copy` requires `Clone`, even if cloning is as cheap as copying

Reply:
I meant both, but it does not hurt to have them :)

Reply:
Ok well then I don't get your comment @Empty2k12